janitor/functions/read_commandline.py

```diff
@@ -0,0 +1,77 @@
+import pandas_flavor as pf
+import pandas as pd
+from subprocess import Popen, PIPE
+import sys
+
+
+@pf.register_dataframe_method
+def read_commandline() -> pd.DataFrame:
+    """
+    Read command from commandline.
+
+    argv[0] should be program name
+    argv[1:] will be the a list of commands available to the shell
+    such as:
+    cat sep-quarter.csv | grep .SEA1AA
```

Comment:
```suggestion
    For example, you may wish to run the following command on `sep-quarter.csv`
    before reading it into a pandas DataFrame:
    
    ```bash
    cat sep-quarter.csv | grep .SEA1AA
    ```
```